reference/catalog.v3.yml

```diff
@@ -903,7 +903,8 @@ paths:
         - 250 characters product name length.
 
         **Usage Notes**
-        * `POST` requests to `/products` accepts a single `video` object; to send an array of video objects, see: `/products/{product_id}/videos`.
+        * `POST` requests to `/products` accept a `video` array. 
+        * `POST` requests to `/products/{product_id}/videos` accept a video object.
```

Comment:
```suggestion
        * `POST` requests to `/products/{product_id}/videos` accept a `video` object.
```

Reply:
My only suggestion for the usage notes would be to make it more conversational like:
Send a Post request at `/...` to accept a `video` object. Otherwise, no issues here! 😺

Reply:
good catch 🎈